crates/hyfetch/src/cli_options.rs

```diff
@@ -170,6 +171,10 @@ BACKEND={{{backends}}}",
         .help("Ask for input before exiting")
         .switch()
         .hide();
+    let auto_detect_light_dark = long("auto-detect-light-dark")
+        .help("Enables hyfetch to detect light/dark terminal background in runtime")
+        .argument("BOOL")
```

Comment:
This should be something like `"DETECT_BG"`

https://docs.rs/bpaf/latest/bpaf/parsers/struct.NamedArg.html#method.argument

> For `metavar` value you should pick something short and descriptive about the parameter, usually in capital letters. For example for an abstract file parameter it could be `"FILE"`, for a username - `"USER"`, etc.